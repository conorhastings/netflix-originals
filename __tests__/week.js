import React from 'react';
import renderer from 'react-test-renderer';
import Week from '../src/components/week.js';


test('Week component renders without error', () => {
  const tree = renderer.create(
    <Week days={[1,2,3,4,5,6,7]} viewHeight={12} heightInPx={77} launches={{}} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});