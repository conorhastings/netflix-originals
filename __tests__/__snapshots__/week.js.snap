// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Week component renders without error 1`] = `
<div
  className="css-yluy4z"
>
  <div
    className="css-1vew5v7"
  >
    <div
      className="css-1i9y2d3"
    >
      1
    </div>
  </div>
  <div
    className="css-1vew5v7"
  >
    <div
      className="css-1i9y2d3"
    >
      2
    </div>
  </div>
  <div
    className="css-1vew5v7"
  >
    <div
      className="css-1i9y2d3"
    >
      3
    </div>
  </div>
  <div
    className="css-1vew5v7"
  >
    <div
      className="css-1i9y2d3"
    >
      4
    </div>
  </div>
  <div
    className="css-1vew5v7"
  >
    <div
      className="css-1i9y2d3"
    >
      5
    </div>
  </div>
  <div
    className="css-1vew5v7"
  >
    <div
      className="css-1i9y2d3"
    >
      6
    </div>
  </div>
  <div
    className="css-1vew5v7"
  >
    <div
      className="css-1i9y2d3"
    >
      7
    </div>
  </div>
</div>
`;
